lib/downloader.cjs

```diff
@@ -0,0 +1,221 @@
+/**
```

Comment:
Looks like this file got moved to another repo, will delete it from this branch later, leaving it up in case any changes from it want to be copied to the other repo.

Reply:
You can go ahead and remove this